omod-common/src/main/java/org/openmrs/module/webservices/rest/util/ReflectionUtil.java

```diff
@@ -17,7 +17,13 @@
 import java.lang.reflect.ParameterizedType;
 import java.lang.reflect.Type;
 import java.lang.reflect.TypeVariable;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 
+import com.sun.tools.javac.util.Pair;

```

Comment:
I would have just added a CacheKey class.
But I don't have a big problem importing javatuples, if it's a very small library...